Doc/library/functions.rst

```diff
@@ -164,6 +164,8 @@ are always available.  They are listed here in alphabetical order.
    :func:`sys.breakpointhook` can be set to some other function and
    :func:`breakpoint` will automatically call that, allowing you to drop into
    the debugger of choice.
+   If :func:`sys.breakpointhook` is not available to be called, this function will
```

Comment:
This descriptions sounds like it will raise error when `callable(sys.breakpointhook)` is not true.

But when sys.breakpointhook is set but not callable, it doesn't raise RuntimeError. (e.g. setting it to `1`) RuntimeError is raised only when `getattr(sys, 'breakpointhook')` is False.